rocAL/rocAL_pybind/examples/rocAL_api_pytorch_classification_reader.py

```diff
@@ -68,29 +68,27 @@ def main():
         jpegs, labels = fn.readers.file(file_root=data_path)
         decode = fn.decoders.image_slice(jpegs, output_type=types.RGB,
```

Comment:
Remove any underscores in the variables declared.
    _rali_cpu = True if sys.argv[2] == "cpu" else False